src/audio/AudioResource.ts

```diff
@@ -251,3 +263,57 @@ export function createAudioResource<T>(
 		options.silencePaddingFrames ?? 5,
 	);
 }
+
+export function createFFmpegResource<T>(
+	input: string,
+	options?: createFFmpegResourceOptions,
+): AudioResource<T extends null | undefined ? null : T>;
+
+export function createFFmpegResource(input: string, options: createFFmpegResourceOptions = {}): AudioResource | void {
+	const final_args: string[] = [];
+	const FFMPEG_OPUS_ARGUMENTS = [
+		'-analyzeduration',
+		'0',
+		'-loglevel',
+		'0',
+		'-acodec',
+		'libopus',
+		'-f',
+		'opus',
+		'-ar',
+		'48000',
+		'-ac',
+		'2',
+	];
+	const FFMPEG_PCM_ARGUMENTS = [
+		'-analyzeduration',
+		'0',
+		'-loglevel',
+		'0',
+		'-f',
+		's16le',
+		'-acodec',
+		'pcm_s16le',
+		'-ar',
+		'48000',
+		'-ac',
+		'2',
+	];
```

Comment:
These two constants are copied from elsewhere in the project. Can you make sure they're defined in one place and then imported here instead.